openwisp-monitoring/files/monitoring.agent

```diff
@@ -118,6 +118,7 @@ handle_sigusr1() {
 }
 
 send_data() {
+	success=1
```

Comment:
After having reviewed the code I think you were right in setting this to zero, my calculation was incorrect.